content/configuration/source.md

```diff
@@ -83,18 +89,19 @@ communications:
                         sources:
                             - first-source
                             - second-source
+                            - third-source
 ```
 
 In a result, for the `random` channel, the merged recommendation configuration is as follows:
 
 ```yaml
 recommendations:
     pod:
-        noLatestImageTag: false # none of the sources has this property set to true
-        labelsSet: true
+        noLatestImageTag: false # set in `first-source` to `false`, overriden in `third-source`
```

Comment:
`overriden in ` but with the same value - it would be nice to add that as I was looking how it was "overridden" and didn't know if it is a bug with YAML or not